docs/src/prototypes/alerts/BannerAlerts.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react'
+import * as _ from 'lodash'
+import {
+  Flex,
+  Provider,
+  RadioGroup,
+  ThemeAnimation,
+  RadioGroupItemProps,
+  Header,
+  Button,
+} from '@stardust-ui/react'
+
+import AnimatedBannerAlert from './AnimatedBannerAlert'
+import ComposeMessage from '../chatPane/composeMessage'
+
+type BannerName = 'info' | 'oof' | 'danger' | 'urgent' | 'critical'
+
+const isAlertClosable = (bannerName: BannerName) =>
+  bannerName !== 'critical' && bannerName !== 'urgent'
+
+const getBannerContent = (bannerName: BannerName) => `${_.startCase(bannerName)} banner`
+
+const bannerNames: BannerName[] = ['info', 'oof', 'danger', 'urgent', 'critical']
+
+const bannerRadioItems: RadioGroupItemProps[] = bannerNames.map(bannerName => ({
+  key: bannerName,
+  value: bannerName,
+  label: getBannerContent(bannerName),
+}))
+
+const slideDown: ThemeAnimation = {
+  keyframe: {
+    from: { transform: 'translateY(0)' },
+    to: { transform: 'translateY(100%)', display: 'none' },
+  },
+  duration: '.3s',
+  iterationCount: '1',
+  timingFunction: 'linear',
+  fillMode: 'forwards',
+}
+
+interface BannerAlertsState {
+  selectedBannerName: string
```

Comment:
```suggestion
  selectedBannerName: BannerName 
```

This allow to get rid of `as BannerName` in some places

Reply:
oops missed this, thanks! applied